bounties_api/std_bounties/tests.py

```diff
@@ -300,6 +471,46 @@ def test_extend_deadline(self):
             pk=self.bounty_to_extend_deadline_id)
         self.assertEqual(result, bounty_to_extend_deadline_from_db)
 
+    def test_change_bounty(self):
+        bounty_to_change = Bounty.objects.get(pk=self.bounty_to_change_id)
+        inputs = {
+            'data': 'QmQjchBM6tjAvXzkDEpWgLUv9Ui4jwqtxsEzB6LxB2WqFL',
+            'newDeadline': '1517536923',
+            'newFulfillmentAmount': 2,
+            'newArbiter': '0x4444444444444444444422222222222222222222'
+        }
+        deadline = datetime.fromtimestamp(int(inputs['newDeadline']))
+        result = self.client.change_bounty(
+            bounty=bounty_to_change,
+            inputs=inputs)
+        self.assertEqual(result.deadline, deadline)
+        self.assertEqual(result.fulfillmentAmount, inputs['newFulfillmentAmount'])
+        self.assertEqual(result.arbiter, inputs['newArbiter'])
+
+    def test_transfer_issuer(self):
+        bounty_to_transfer_issuer = Bounty.objects.get(pk=self.bounty_to_transfer_issuer_id)
+        inputs = {
+            'newIssuer': '0x4444444444444444444422222222222222222222'
+        }
+        result = self.client.transfer_issuer(
+            bounty=bounty_to_transfer_issuer,
+            inputs=inputs)
+        self.assertEqual(result.issuer, inputs['newIssuer'])
+
+    def test_increase_payout(self):
+        bounty_to_increase_payout = Bounty.objects.get(pk=self.bounty_to_increase_payout_id)
+        old_balance = bounty_to_increase_payout.balance
+        inputs = {
+            'value': 20,
```

Comment:
Could you also test this without changing the value?

Reply:
Good idea.